configure.py

```diff
@@ -688,6 +688,11 @@ def find_ninja():
 arg_parser = argparse.ArgumentParser('Configure scylla')
 arg_parser.add_argument('--out', dest='buildfile', action='store', default='build.ninja',
                         help='Output build-file name (by default build.ninja)')
+arg_parser.add_argument('--out-actual-filename', dest="buildfile_actual_filename", action='store',
```

Comment:
I'm open to suggestions/bikeshedding about the name, I don't like it too much but don't have better ideas right now.

Reply:
We shouldn't have --out or --our-actual-filename (or --out-this-time-its-really-the-filename). It's okay for ./confgure.py to write to a temporary file, but it should rename the temporary file back to build.ninja when it's done.

Reply:
See https://github.com/scylladb/scylladb/commit/8db9e62de92945b4322772aa95715ad4277f2a87 for why "--out" was introduced. The point is that we want the caller, not the callee, to do the eventual renaming, because when the callee did the renaming, it was vulnerable to interruptions. Still, the "--out" was supposed to name a fixed-name temporary file, it wasn't supposed to be calculated and become something like build.ninja.new.new.new. It was fine after commit 8db9e62de92945b4322772aa95715ad4277f2a87 but got broken in https://github.com/scylladb/scylladb/commit/be3776ec2ab11e0c13f191ec79e907b358a95612.

Reply:
In the comment below I called the "out-actual-filename" the "final resting place" ;-) Maybe "--out-final-name" will be more self-explanatory than out-"actual"(?)-name.

Reply:
I agree that --out-final-name looks better than --out-actual-filename, I'll change it.